packages/application-extension/src/index.tsx

```diff
@@ -328,6 +362,38 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.toggleSidebarWidget, {
+        label: trans.__('Toggle Sidebar Element'),
+        caption: trans.__('By default opens first element of the sidebar'),
```

Comment:
This is not the case I think? By default it errors as `args.index` is `undefined`?

```suggestion
```

Reply:
Also, caption may not be the best place to mention it even if it were working.

Reply:
@brichet good to merge once this thread is addressed. Feel free to go either way.

Reply:
I have removed the caption